homeassistant/components/unifi/switch.py

```diff
@@ -284,3 +338,73 @@ async def options_updated(self) -> None:
         """Config entry options are updated, remove entity if option is disabled."""
         if self.client.mac not in self.controller.option_block_clients:
             await self.remove_item({self.client.mac})
+
+
+class UniFiDPIRestrictionSwitch(CoordinatorEntity, UniFiBase, SwitchEntity):
+    """Representation of a DPI restriction group."""
+
+    DOMAIN = DOMAIN
+    TYPE = DPI_SWITCH
+
+    def __init__(self, item, controller, coordinator) -> None:
+        """Set up DPI switch."""
+        CoordinatorEntity.__init__(self, coordinator)
+        UniFiBase.__init__(self, item, controller)
+
+    @property
+    def key(self) -> Any:
```

Comment:
Shouldn't it always be a string?
```suggestion
    def key(self) -> str:
```

Reply:
Today with `mac`, `_id`, and `name` as the only keys, it is. But there's nothing really that says it **needs** to be a string. If later we use one of the non-string attributes as the key.

Reply:
Ok